src/App.tsx

```diff
@@ -15,37 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  SORT_BY_CHAR = 'SORT_BY_CHAR',
+  SORT_BY_LENGTH = 'SORT_BY_LENGTH',
+}
+
+type TGoods = string[];
+
+interface IGoodsProps {
+  goods: Readonly<TGoods>;
+}
+
+const Goods: FC<IGoodsProps> = ({ goods }) => (
+  <ul>
+    {goods.map(el => (
+      <li data-cy="Good" key={el}>
+        {el}
+      </li>
+    ))}
+  </ul>
+);
+
+type TSortGoods = (
+  goods: TGoods,
+  sortType: typeof SortType | string,
+  isReverse: boolean,
+) => TGoods;
+
+const sortGoods: TSortGoods = (goods, sortType, isReverse = false) => {
```

Comment:
move it ti another directory - `utils/sortGoods/sortGoods.ts`

Reply:
I think there is no difference and for me using types for functions is more understandable and readable